src/web/resources/report.js

```diff
@@ -86,21 +93,51 @@ const ClientGraph = new Component('#graphs', {
     },
 
     render_functions: function($elt, selected_var_name, selected_functions) {
-        const all_lines = ALL_LINES.filter(o => this.points_json.error[o.name] != false)
         const toggle = (option, options) => options.includes(option) ? options.filter(o => o != option) : [...options, option]
-        $elt.replaceChildren.apply(
-            $elt,
-            all_lines.map(fn => 
-                Element("label", [
+
+        // this.points_json.error is a dictionary where keys are indices "1", "2", ..., "n".
+        // The values for each key is broken up into two parts : error_type and actual error values
+        // error_type is the first element in the list value with 1...n being the remianing values
+        // Types of error type is "start", "end", "target1", "target2", ..., "targetm" 
+
+        var curr_list = []
+        let i = 0
+
+        for (const key in this.points_json.error) {
+            if (this.points_json.error[key][1] !== false) {
```

Comment:
Again, if we didn’t add the #f entry we wouldn’t need this test.

Reply:
Removed!